### ISSUE TITLE
Privacy Center Consent Demo page should not be enabled by default

### ISSUE THREAD

**Issue by mfbrown** [ethyca/fides#2955]
We offer a nifty privacy center consent demo page where you can see the values of the consent object at /fides-consent-demo.html off the privacy center.

The only problem is that it's always on. Can we turn it off by default or with a dev environment tag and allow people to turn it on to test?

**Comment by ThomasLaPiana** [comment]
@allisonking I want to get my hands dirty with more FE work, can you point me in the _general_ direction of how I might want to implement this and then I can eventually stumble to the right solution?

**Comment by allisonking** [comment]
hmmm this is kind of a tricky thing since `fides-consent` is quite different than the rest of our packages (and is also currently going through [big changes](https://github.com/ethyca/fides/pull/3135)!) is it possible to wait on this ticket until some of the bigger turborepo/fides-consent changes are in? we're changing how we build fides-consent, so that'll have pretty big implications on this ticket.

in general though, it looks like there's a ` fides-consent-demo.html` page in the privacy-center (though it looks like it's getting moved to `fides-consent`, newly named `fides-js`, in Neville's branch). That's the page we want to either show or not show.

so we need:
1. a way to say whether or not we want this to show. we could usually use environment variables, but since this is just an HTML page, I don't think it'll have access to our next/node environment variables. it does have access to the `window.Fides` object, so we might be able to pass something in there. this might depend on whether we need to be able to configure show/not show at build time or run time
2. a way to not show the page. the "proper" thing would probably be to, when the user builds `fides-consent`, to check some sort of flag, and if that flag says don't show, then we don't bundle the html page with the package (this is assuming it's ok that this is configured at build time). otherwise, we could still bundle in `fides-consent-demo.html`, but within the html have a `<script>` tag which checks for the flag in step 1, then if it says don't show, redirects you somewhere else (with something like a `window.location.href = window.location.origin`)

Also, building `fides-consent` is a little weird, and is the big thing that's changing right now. so it'll be a bit of an obstacle to get this running/testable 

all to say, it would be easier to work on this after the turborepo/fides-consent refactor, but also it won't be as easy as it would be in i.e. admin-ui or privacy-center since `fides-consent` doesn't have all the bells and whistles of a full app

**Comment by ThomasLaPiana** [comment]
@allisonking thank you for the in-depth comment! I'll bump this out and we can take another look when things are more stable 😄

**Comment by mfbrown** [comment]
We should probably just turn this page off for now, since it's not really going to work with the new consent stuff. It's definitely a one off thing though and isn't explicitly part of the consent epics.

**Comment by ThomasLaPiana** [comment]
@allisonking @eastandwestwind has this been removed already by any chance?

**Comment by rsilvery** [comment]
@allisonking  how does this fit into our new world - is it the fides-js-demo page?

**Comment by allisonking** [comment]
Yup that's the page. the issue is still valid—anyone could hit that demo page on a running privacy center
